mailer/mailer.go

```diff
@@ -78,7 +79,7 @@ func (m *mailer) SendEmail(subject, content string, recipients ...string) error
 		return err
 	}
 
-	err = smtp.SendMail(server, m.auth, m.user, recipients, msg.Bytes())
+	err = smtp.SendMail(server, m.auth, m.from, recipients, msg.Bytes())
```

Comment:
@AlexMichaelJonesNC here is one change I'm curious about. Wonder if the `from` parameter in the `SendEmail` call should be `m.from` or the actual username. In case this doesn't work, we will have to re add the `m.user` filed in the mailer struct and change this line to use `m.user` instead of `m.from`

Reply:
I think it should be m.from, as we discussed in slack mu.user doesn't work for all vendors - example: for AWS the from address must be a properly formatted email address that is on your outbound whitelist, since the user in that context is just a secret key id, it fails that validation and mail is rejected.